pep-9999.rst

```diff
@@ -0,0 +1,204 @@
+
+To minimize the footprint further, a range of ideas was considered, see `Python minimization in Fedora`_. These ideas have been then discussed on this `Fedora devel mailing list thread`_. Here's a summary of the proposed solutions and a brief explanation why they were rejected for this PEP.
+ - Solution 1: Slim down the Python standard library — this is out of scope for this PEP
+ - Solution 2: Move developer oriented modules to python3-devel (or split the stdlib into pieces) — this breaks users expectations of "batteries included Python"
+ - Solution 3: Compress large data-like modules — already implemented in Fedora, but does not lead to sufficient minimization
+ - Solution 4: ZIP the entire standard library — this would break users expectations, lead to significant divergence between normal and constrained environments
+ - Solution 5: Stop shipping mandatory bytecode cache — part of the solutions proposed in this PEP
+ - Solution 6: Stop shipping mandatory optimized bytecode cache — part of the solutions proposed in this PEP
+ - Solution 7: Stop shipping mandatory source files, ship .pyc instead — part of the solutions proposed in this PEP
+ - Solution 8: Compress .pyc files — Impacts performance, does not lead to sufficient minimization
+ - Solution 9: Deduplicate bytecode cache — already implemented in Fedora, but does not lead to sufficient minimization
+ - Solution 10: Stop shipping mandatory Python, rewrite dnf to Rust — irrelevant to the Python ecosystem
+
+`Compressing .pyc bytecode files`_ has already been discussed, but leads to slower startup times due to decompression.
+
+Open Issues
+===========
+
+The exact method to accomplish the minimization has not yet been settled on. We propose two base methods that we feel represent the best ways forward. Whether one of those, a combination of those, or perhaps a yet different solution is chosen remains an open issue.
+
+
+References
+==========
+.. _`Option 1: Shipping only .py files, and disabling creation of .pyc files`:
+   https://github.com/encukou/cpython/tree/minimization-marker
+.. _`Option 2: Shipping the non-optimized .pyc files and compressed .py source files`:
+   https://github.com/encukou/cpython/tree/minimization-src-from-pyc
+
+.. _`python3-libs`:
+   https://src.fedoraproject.org/rpms/python3.9
+
+.. _`Python minimization in Fedora`:
+   https://github.com/hroncok/python-minimization/blob/master/document.md
```

Comment:
If we want to link this from the PEP, I'd rather tag it somehow, so it doesn't change.

Reply:
Good idea. I can tag it using a commit. But I'd say a tag would be better (just in case we need to change it for some reason). What do you think?

Reply:
@hroncok I assumed you'd do the tag, is that correct? I'm not sure if I can open a PR to push a tag to your repo.